### PR TITLE
Improvements in prompt text

### LINKED ISSUE #564: Improvements in prompt text when running Livepeer for the first time and setting passphrase
**Is your feature request related to a problem? Please describe.**

When showing a new user how to run `livepeer`, I become aware of a lack of clarity in the instructions given to them in Terminal. In some cases, I believe we could do better at explaining what a user must do at each step of the process.

This improvement is designed to attempt to clarify exactly what a user is supposed to do at every step of the process to set up a Transcoder node.

**Describe the solution you'd like**

The first time `./livepeer` is run in Terminal, the text shown to a user is:

```
chrishobcroft@psgex8917:~$ ./livepeer_linux/livepeer -transcoder
I0920 07:29:35.144881    2646 livepeer.go:110] ***Livepeer is running on the Ethereum main network: 0xf96d54e490317c557a967abfa5d6e33006be69b3***
I0920 07:29:35.144950    2646 livepeer.go:119] Creating data dir: /home/chrishobcroft/.lpData
I0920 07:29:35.640527    2646 accountmanager.go:41] Please create a new ETH account
Passphrase: 
```

I propose the following:

```
chrishobcroft@psgex8917:~$ ./livepeer_linux/livepeer -transcoder
I0920 07:29:35.144881    2646 livepeer.go:110] ***Livepeer is running on the Ethereum main network: 0xf96d54e490317c557a967abfa5d6e33006be69b3***
I0920 07:29:35.144950    2646 livepeer.go:119] Creating data dir: /home/chrishobcroft/.lpData
I0920 07:29:35.640527    2646 accountmanager.go:41] This process will create a new Ethereum account for this Livepeer node
I0920 07:29:35.640527    2646 accountmanager.go:41] Please enter a passphrase to encrypt the Private Keystore file for the Ethereum account.
I0920 07:29:35.640527    2646 accountmanager.go:41] This process will ask for this passphrase every time it is launched.
I0920 07:29:35.640527    2646 accountmanager.go:41] (no characters will appear in Terminal when the passphrase is entered)
Passphrase: 
```

Furthermore, after you have entered the passphrase twice, the Terminal currently displays:

```
Repeat passphrase: 
I0920 07:38:13.229920    2646 accountmanager.go:58] Using ETH account: 0x5e0D6Db0DCe1ef7413eF9ff39b02449eF443c868
I0920 07:38:14.380987    2646 accountmanager.go:76] Passphrase required to unlock ETH account 0x5e0D6Db0DCe1ef7413eF9ff39b02449eF443c868
Passphrase: 
```

I would propose adding a line to explain what has happened in the background after "Repeat passphrase":

```
Repeat passphrase: 
I0920 07:38:13.229920    2646 accountmanager.go:58] Creating data file: /home/chrishobcroft/.lpData/keystore/UTC--2018-09-20T11-38-12.201213658Z--5e0d6db0dce1ef7413ef9ff39b02449ef443c868
I0920 07:38:13.229920    2646 accountmanager.go:58] Using Ethereum account: 0x5e0D6Db0DCe1ef7413eF9ff39b02449eF443c868
I0920 07:38:14.380987    2646 accountmanager.go:76] Please enter the passphrase to unlock Ethereum account 0x5e0D6Db0DCe1ef7413eF9ff39b02449eF443c868
Passphrase: 
```

**Describe alternatives you've considered**

We could continue to display the current text, and assume that people who are running the software will have sufficient skill and experience to understand what is going on.

**Additional context**

I have set up a lot of Transcoder nodes, and I am also not a Livepeer Core Developer - so I feel like this gives me more of an "outsider's" perspective in terms of outlook on descriptions.

### PR DESCRIPTION
**Pull request by samparsky**
**What does this pull request do? Explain your changes. (required)**
This improvement is designed to attempt to clarify exactly what a user is supposed to do at every step of the process to set up a Transcoder node.

**Specific updates (required)**
- Makes changes to the description text in setting up a livepeer node 

**How did you test each of these updates (required)**
Checked the output

**Does this pull request close any open issues?**
Fixes #564 

**Checklist:**
- [ ] README and other documentation updated
- [x] Node runs in OSX and devenv
- [x] All tests in `./test.sh` pass

### CODE REVIEW

File: eth/accountmanager.go [hunk 1]
```diff
@@ -38,8 +38,11 @@ func NewAccountManager(accountAddr ethcommon.Address, keystoreDir string) (*Acco
 	var acct accounts.Account
 	var err error
 	if numAccounts == 0 || ((accountAddr != ethcommon.Address{}) && !acctExists) {
-		glog.Infof("Please create a new ETH account")
-
+		glog.Info("This process will create a new Ethereum account for this Livepeer node")
```
Comment (j0sh) [inline eth/accountmanager.go]: Perhaps something like: "No Ethereum account found. Creating a new account."

**Review by j0sh — CHANGES_REQUESTED** [review]
Generally looks good, left a small suggestion on wording. However, the compiled Livepeer binary is checked in. That should be removed.

File: eth/accountmanager.go (hunk 1 above)
Comment (samparsky) [reply eth/accountmanager.go]: Done

**Review by j0sh — APPROVED** [review]
LGTM, thanks!

**Comment by chrishobcroft** [comment]
Good stuff, thanks!

**Pull request merged**